### ISSUE TITLE
include nextcloud in tooltip for "Allow note file names to be different from headline"

### ISSUE THREAD

**Issue by brainchild0** [pbek/QOwnNotes#1110]
The "General" section of settings has a toggle labelled: "Allow note file names to be different from headline".

The popup text adds the following note:  "Beware of that ownCloud notes needs them to be the same!"

Should this clarify that Nextcloud has the same limitation?  The first time I read the message, I assumed that it was saying that support is available in Nextcloud but not ownCloud, which I believe is inaccurate.

**Comment by brainchild0** [comment]
Correct.  Also, while doing it, you could also correct the English grammar: "Beware that" instead of "Beware of that".

**Comment by pbek** [comment]
## 19.1.1
- changed the tooltip text for the `Allow note file name to be different from headline`
  setting to be more clear

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by pbek** [comment]
Great, thank you for testing!
